extensions/jdbc/jdbc-mariadb/runtime/src/main/java/io/quarkus/jdbc/mariadb/runtime/graal/ConnectionHelper_socketHandler.java

```diff
@@ -1,13 +1,13 @@
 package io.quarkus.jdbc.mariadb.runtime.graal;
 
-import org.mariadb.jdbc.internal.io.socket.SocketHandlerFunction;
+import org.mariadb.jdbc.client.socket.impl.SocketHandlerFunction;
 
 import com.oracle.svm.core.annotate.Alias;
 import com.oracle.svm.core.annotate.RecomputeFieldValue;
 import com.oracle.svm.core.annotate.TargetClass;
 
-@TargetClass(org.mariadb.jdbc.internal.util.Utils.class)
-public final class Utils_socketHandler {
+@TargetClass(className = "org.mariadb.jdbc.client.impl.ConnectionHelper")
```

Comment:
What's the policy for referencing target classes that have `static` fields or initializers?
I wanted to avoid that the class is being initialized here (and went for just the name instead), but I have to admit that I don't know if that's even happening as per JLS.

Reply:
I prefer quoting as well. +1